spec/index.bs

```diff
@@ -813,14 +813,16 @@ dictionary GPUObjectDescriptorBase {
         The initial value of {{GPUObjectBase/label|GPUObjectBase.label}}.
 </dl>
 
-## Invalid Internal Objects &amp; Contagious Invalidity ## {#invalidity}
+## Asynchrony ## {#asynchrony}
 
-Object creation operations in WebGPU are internally asynchronous, so they don't fail with exceptions.
-Instead, returned objects may refer to [=internal objects=] which are either
+### Invalid Internal Objects &amp; Contagious Invalidity ### {#invalidity}
+
+Object creation operations in WebGPU don't return promises, but nonetheless are internally
+asynchronous, and don't fail with exceptions or rejections. Instead, returned objects refer to
+[=internal objects=], which exist on a [=device timeline=] where they are either
 <dfn dfn>valid</dfn> or <dfn dfn>invalid</dfn>.
-An [=invalid=] object may never become [=valid=] at a later time.
-Some objects may become [=invalid=] during their lifetime, while most may only
-be [=invalid=] from creation.
+An [=invalid=] object may never become [=valid=] at a later time,
```

Comment:
```suggestion
An [=invalid=] object will never become [=valid=] at a later time,
```

Silly grammer nit, but on my first reading of this it sounded to me like "may" left wiggle room for some invalid object to become valid.